---
title: "Emergence of the B.1.214.2 SARS-CoV-2 Lineage with an Omicron-Like Spike Insertion and a Unique Upper Airway Immune Signature"
collection: contributions
category: manuscripts
permalink: /contributions/2023-holtz-b12142
date: 2023-12-11
venue: 'SSRN'
paperurl: 'https://ssrn.com/abstract=4650308'
citation: 'Holtz A, Van Weyenbergh J, Hong SL, Cuypers L, O&apos;Toole √Å, <b>Dudas G</b>, Gerdol M, Potter BI, Ntoumi F, Mapanguy CCM, Vanmechelen B, Wawina-Bokalanga T, Van Holm B, Menezes SM, Soubotko K, Van Pottelbergh G, Wollants E, Vermeersch P, Jacob AS, Maes B, Obbels D, Matheeussen V, Martens G, Gras J, Verhasselt B, Laffut W, Vael C, Goegebuer T, van der Kant R, Rousseau F, Schymkowitz J, Serrano L, Delgado J, Wenseleers T, Bours V, Andre E, Suchard MA, Rambaut A, Dellicour S, Maes P, Durkin K, Baele G, 2023. &quot;Emergence of the B.1.214.2 SARS-CoV-2 Lineage with an Omicron-Like Spike Insertion and a Unique Upper Airway Immune Signature&quot;. <i>Preprints with the Lancet</i> https://ssrn.com/abstract=4650308.'
doi: 10.2139/ssrn.4650308
tags:
  - SARS-CoV-2
  - genomic epidemiology
  - phylogeography
---

### Background
Since 2020, SARS-CoV-2 has rapidly accumulated mutations that have accelerated viral spread. First identified in January 2021 in Belgium, the unique B.1.214.2 variant exhibited a 3-amino acid insertion in the Spike protein, similar to the later Omicron BA.1 variant. This insertion was speculated to confer an evolutionary advantage, possibly enhancing viral transmissibility or evading immune response.

### Methods
Publicly available (GISAID) B.1.214.-derived whole genome sequences were added to a NextStrain build of Central African and global sequences (12/28/2019-07/08/2021, total n=3,507), followed by travel history-aware Bayesian phylogeographic analysis. Protein modeling was used to predict Spike three-dimensional structure. Serum neutralizing antibodies were quantified by competition with ACE2/Spike binding of 10 SARS-CoV2 variants (MSD). Immunological profiling was performed by digital transcriptomics (Nanostring).

### Findings
First detected in returning travelers, B.1.214.2 exhibited a significant uneven presence across Central Africa, Belgium, Switzerland, and France, peaking during April 2021. Travel history-aware phylogeographic analysis estimated the Republic of the Congo its origin to be, with a main introduction to Europe via France and Belgium, and several smaller introductions across the epidemic period, thus. We modeled the conformation of the insertion to the Spike protein exterior but outside the ACE2 interaction site. Nationwide monitoring of SARS-CoV-2 nursing home outbreaks revealed a moderately severe clinical phenotype (8.7% case fatality ratio). In these high-risk elderly, we found no difference in neutralizing antibodies but a quantitatively (80% unique) and qualitatively distinct (higher B- and T-cell activation, type I IFN signaling and lower NK, Th17, and complement system activation) nasopharyngeal immune signature in B.1.214.2 infection, as compared to similar outbreaks.

### Interpretation
We link B.1.214.2 global spread to human travel patterns. The Spike insertion outside the ACE2 binding site does not confer antibody escape but a unique upper airway immune signature may help explain the peculiar epidemiological pattern of the B.1.214.2 lineage.