---
title: "Emergence and spread of SARS-CoV-2 lineage B.1.620 with variant of concern-like mutations and deletions"
collection: publications
category: manuscripts
permalink: /publication/2021-sarscov2-b1620
date: 2021-10-01
venue: 'Nature Communications'
paperurl: 'https://www.nature.com/articles/s41467-021-26055-8'
localurl: 'http://evogytis.github.io/files/2021-sarscov2-b1620.pdf'
citation: '<b>Dudas G</b>, Hong SL, Potter B, Calvignac-Spencer S, Niatou-Singa FS, Tombolomako TB, Fuh-Neba T, Vickos U, Ulrich M, Leendertz FH, Khan K, Huber C, Watts A, Olendraitė I, Snijder J, Wijnant KN, Bonvin AMJJ, Martres P, Behillil S, Ayouba A, Maidadi FM, Meta Djomsi D, Godwe C, Butel C, Šimaitis A, Gabrielaitė M, Katėnaitė M, Norvilas R, Raugaitė L, Koyaweda G, Kaleb Kandou J, Jonikas R, Nasvytienė I, Žemeckienė Ž, Gečys D, Tamušauskaitė K, Norkienė M, Vasiliūnaitė E, Žiogienė D, Timinskas A, Šukys M, Šarauskas M, Alzbutas G, Amuri A, Lusamaki E, Makangara J, Muyembe F, Lofiko E, Mbala P, Tamfum J, Darnycka Belizaire M, Essomba R, Assoumou M, Mboringong A, Dieng A, Juozapaitė D, Hosch S, Obama J, Ondo’o Ayekaba M, Naumovas D, Pautienius A, Rafaï C, Vitkauskienė A, Ugenskienė R, Gedvilaitė A, Čereškevičius D, Lesauskaitė V, Žemaitis L, Griškevičius L, Baele G, 2021. &quot;Emergence and spread of SARS-CoV-2 lineage B.1.620 with variant of concern-like mutations and deletions&quot;. <i>Nature Communications</i> 12: 5769.'
doi: 10.1038/s41467-021-26055-8
tags:
  - SARS-CoV-2
  - genomic epidemiology
---

Distinct SARS-CoV-2 lineages, discovered through various genomic surveillance initiatives, have emerged during the pandemic following unprecedented reductions in worldwide human mobility.
We here describe a SARS-CoV-2 lineage - designated B.1.620 - discovered in Lithuania and carrying many mutations and deletions in the spike protein shared with widespread variants of concern (VOCs), including E484K, S477N and deletions HV69Δ, Y144Δ, and LLA241/243Δ.
As well as documenting the suite of mutations this lineage carries, we also describe its potential to be resistant to neutralising antibodies, accompanying travel histories for a subset of European cases, evidence of local B.1.620 transmission in Europe with a focus on Lithuania, and significance of its prevalence in Central Africa owing to recent genome sequencing efforts there.
We make a case for its likely Central African origin using advanced phylogeographic inference methodologies incorporating recorded travel histories of infected travellers.

